packages/teams-js/CHANGELOG.md

```diff
@@ -1,345 +1,245 @@
-## 2.0.0-beta.3
-
-### Breaking Changes
-
-#### Core APIs have been moved and renamed
-
-APIs previously under the `core` namespace, `executeDeepLink` and `shareDeepLink`, have moved.
-
-- `core.executeDeepLink` has renamed and moved to `app.openLink`
-- `core.shareDeepLink` has moved to `pages.shareDeepLink`
-
-### Bug fixes
-
-`null` runtimeConfig is no longer allowed during initialization. This will now throw a "Received runtime config is invalid" error.
-
 ## 2.0.0-beta.2
 
-### Bug Fixes
+### Patches
 
-Update TSDoc @deprecated comments to include links to replaced APIs.
+- Update TSDoc @deprecated comments to include links to replaced APIs.
 
-Update webpack-dev-server types to match webpack 5 versions and stop generating module wrappers in MicrosoftTeams.d.ts.
+- Update webpack-dev-server types to match webpack 5 versions and stop generating module wrappers in MicrosoftTeams.d.ts.
 
-Fix warnings produced during documentation generation, including exporting additional existing interfaces.
+- Fix warnings produced during documentation generation, including exporting additional existing interfaces.
 
-Update repository URLs to reference `2.0-preview` branch.
+- Update repository URLs to reference `2.0-preview` branch.
 
 ## 2.0.0-beta.1
 
-### Bug Fixes
+### Patches
 
-Update integrity hash to valid value in README file.
+- Update integrity hash to valid value in README file.
 
 ## 2.0.0-beta.0
 
-### Breaking changes
-
-#### The Teams JavaScript client SDK repo is now a monorepo
-
-We utilized [Yarn Workspaces](https://classic.yarnpkg.com/en/docs/workspaces/) to turn our repo into a monorepo. The files specific to the Teams client SDK have been moved to an inner directory
-with the name `teams-js`. A new TeamsJS Test App for validating the Teams client SDK has been added in the <root>/apps/teams-test-app location.
-
-#### Several API functions have been moved and renamed
-
-Organized top-level library functions under a core namespace. For example, `shareDeepLink` has been moved under `core` namespace.
-Using `import * as ... from ...` will now fail. Importing now follows the following convention:
-
-```typescript
-import { core } from '@microsoft/teams-js';
-```
-
-For more detailed API organization, please refer to the **Capabilities organization introduced** section below.
+### Major changes
 
-#### Support for `hostName` added to Context interface
+- The Teams JavaScript client SDK repo is now a monorepo
+- We utilized [Yarn Workspaces](https://classic.yarnpkg.com/en/docs/workspaces/) to turn our repo into a monorepo. <br> The files specific to the Teams client SDK have been moved to an inner directory with the name `teams-js`
+- A new TeamsJS Test App for validating the Teams client SDK has been added in the <root>/apps/teams-test-app location.
```

Comment:
I think these should be indented since "The Teams JavaScript client SDK repo is now a monorepo" is essentially a heading

Reply:
Fixed